apps/artemis-web/pages/api/hijack.ts

```diff
@@ -9,15 +9,26 @@ import { csrf } from '../../libs/csrf';
 
 const handler = nc()
   .use(auth)
-  .use(authorization(['admin']))
+  .use(authorization(['admin', 'user']))
   .post(async (req: NextApiRequestExtended, res: NextApiResponseExtended) => {
     const host: string = process.env.DATABASE_HOST;
     const configHost: string = process.env.CONFIG_HOST;
     const port: number = parseInt(process.env.API_PORT, 10);
     let resp = null;
+    const allowed_normal_actions = ['hijack_action_acknowledge', 'hijack_action_acknowledge_not'];
 
     switch (req.body.action) {
       case 'comment':
+        if (req.user.role !== 'admin') {
```

Comment:
I tried to save a comment with simple user but nothing happens. Please check that there is some feedback on this. Same for the rest of the actions. I can demonstrate during our call too.

Reply:
This works (tested). Resolving.